.github/workflows/main.yml

```
@@ -30,16 +34,23 @@ jobs:
       - run: go generate ./openapi
       - run: go generate -run "wire" ./...
       - name: Auto-commit generated files
+        id: auto_commit
         if: github.event_name == 'pull_request' && github.actor != 'github-actions[bot]' && github.event.pull_request.head.repo.full_name == github.repository
         run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)